packages/airnode-admin/src/cli.ts

```diff
@@ -530,6 +530,28 @@ yargs
       );
     }
   )
+  .command(
+    'generate-mnemonic',
+    'Generates a random mnemonic. Uses "ethers.Wallet.createRandom" under the hood',
+    async () => {
+      const mnemonic = await admin.generateMnemonic();
+      const address = await admin.deriveAirnodeAddress(mnemonic);
+      const xpub = admin.deriveAirnodeXpub(mnemonic);
+
+      logger.log(
+        [
+          'This mnemonic is created locally on your machine using "ethers.Wallet.createRandom" under the hood.',
+          'Make sure to back it up securely, e.g., by writing it down on a piece of paper:',
+          '',
+          ...encaseMnemonic(mnemonic),
+          '',
+          `The default wallet address (path:m/44'/60'/0'/0/0) for this mnemonic is: ${address}`,
+          `The default wallet xpub (path:m/44'/60'/0'/0/0) for this mnemonic is: ${xpub}`,
```

Comment:
Do we need xpub mentioned here? It feels a bit extraneous.

Reply:
I thought about leaving it in since it could be useful since it's also included for the other command, but I can remove if you'd prefer

Reply:
Probably @bbenligiray is best to answer.

Reply:
xpub is probably not needed here